operating_unit/README.rst

```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/license-LGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl.html
+   :alt: License: LGPL-3
+
+==============
+Operating Unit
+==============
+
+An operating unit (OU) is an organizational entity part of a company, with
+separate management ownership. Management by OU is aimed to introduce the
+following features:
+
+* Data partitioning from other OU.
+* Own sequencing schemes.
+* User access administration for data processing and reporting.
+* Product and customer independence of OU.
```

Comment:
Looks good to me "independent of" but not sure "something independence of something else" (rather have from or per).
Just a detail ! and not being a native speaker, I let you make the decision :)